packages/babel-parser/test/fixtures/experimental/import-reflection/invalid-module-source/options.json

```diff
@@ -0,0 +1,3 @@
+{
+  "throws": "Unexpected token, expected \"{\" (1:14)"
```

Comment:
This error isn't the best, since `{` is not allowed here, but we can iterate on it.